usecases/backup/restorer.go

```diff
@@ -70,11 +73,22 @@ func (r *restorer) restoreAll(ctx context.Context,
 	return nil
 }
 
+func getType(myvar interface{}) string {
+	if t := reflect.TypeOf(myvar); t.Kind() == reflect.Ptr {
+		return "*" + t.Elem().Name()
+	} else {
+		return t.Name()
+	}
+}
+
 func (r *restorer) restoreOne(ctx context.Context,
 	pr *models.Principal, backupID string,
 	desc *backup.ClassDescriptor,
 	store objectStore,
 ) (err error) {
+	timer := prometheus.NewTimer(monitoring.GetMetrics().BackupRestoreDurations.WithLabelValues(getType(store.BackupBackend), desc.Name))
```

Comment:
WithLabelValues might panic and we don't want to panic because of a metric.
You could use GetMetricWithLabelValues instead